static/js/click_handlers.js

```diff
@@ -230,6 +233,18 @@ exports.initialize = function () {
         e.stopPropagation();
         popovers.hide_all();
     });
+    $('body').on('click', '.message_edit_notice', function (e) {
+        var row = current_msg_list.get_row(rows.id($(this).closest(".message_row")));
+        current_msg_list.select_id(rows.id(row));
+        var message = current_msg_list.get(rows.id(row));
+        var message_history_cancel_btn = $('#message-history-cancel');
+
+        popovers.hide_actions_popover();
```

Comment:
This should probably be `popovers.hide_all()` since this click handler isn't called from the actions popover.